backend/capellacollab/projects/toolmodels/modelsources/git/handler/exceptions.py

```diff
@@ -39,7 +39,7 @@ async def no_matching_git_instance_handler(
     return await exception_handlers.http_exception_handler(
         request,
         fastapi.HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status_code=status.HTTP_409_CONFLICT,
```

Comment:
404 if there is no matching instance. Conflict would mean that there are two matching instances from my understanding.